### PR TITLE
add invalid response from helios to log

### PR DESCRIPTION
**Pull request by sqreek**
Add invalid response and status code to get more data while investigating HeliosException's Tickets

ping @macbre @ArturKlajnerok

### CODE REVIEW

File: lib/Wikia/src/Service/Helios/HeliosClientImpl.php [hunk 1]
```diff
@@ -91,10 +91,12 @@ public function request( $resourceName, $getParams = [], $postData = [], $extraR
 
 		$this->status = $request->status;
 
-		$output = json_decode( $request->getContent() );
+		$response = $request->getContent();
+		$output = json_decode( $response );
 
 		if ( !$output ) {
-			throw new ClientException ( 'Invalid response.' );
+			$data[ "response" ] = $response;
```
Comment (macbre) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php:99]: Is `$data` variable initialized as an array above?

File: lib/Wikia/src/Service/Helios/HeliosClientImpl.php [hunk 2]
```diff
@@ -91,10 +91,12 @@ public function request( $resourceName, $getParams = [], $postData = [], $extraR
...
+			throw new ClientException ( 'Invalid Helios response.', 0, null, $data );
```
Comment (macbre) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php:101]: Let's log not only the response content, but the response code and `X-Served-By` response header (if Helios sets it) as well

File: lib/Wikia/src/Service/Helios/HeliosClientImpl.php [hunk 3]
```diff
@@ -91,10 +91,14 @@ public function request( $resourceName, $getParams = [], $postData = [], $extraR
...
-		$output = json_decode( $request->getContent() );
+		$response = $request->getContent();
+		$output = json_decode( $response );
 
 		if ( !$output ) {
-			throw new ClientException ( 'Invalid response.' );
+			$data = [];
+			$data[ "response" ] = $response;
+			$data["status_code"] = $request->getStatus();
+			throw new ClientException ( 'Invalid Helios response.', 0, null, $data );
```
Comment (sqreek) [inline lib/Wikia/src/Service/Helios/HeliosClientImpl.php:101]: Message was changed for Jira reporter filter

**Comment by ArturKlajnerok** [comment]
Lgtm :+1:

**Comment by ArturKlajnerok** [comment]
We don't have `X-Served-By` in helios.

**Pull request merged**